src/components/LoginForm/styles.css

```
@@ -1,20 +1,20 @@
-.loginForm  {
-    display:flex;
-    flex-direction: column;
-    align-items:  center;   
+.loginForm {
+	align-items: center;
```

A indentação ta com 4 espaços aqui
```suggestion
  align-items: center;
```